src/views/SubjectAllQuestions/CreateExamModal/CreateExamModal.tsx

```diff
@@ -96,6 +110,15 @@ export const CreateExamModal = ({
               }}
               value={percentage}
             />
+            <div className={classes.learntQuestionsWrapper}>
+              <Checkbox
+                checked={filterLearnt}
+                onChange={(e) => setFilterLearnt(e.target.checked)}
+                className={classes.learntQuestionsCheckbox}
+              />
+              Pomiń nauczone pytania
```

Comment:
It sounds a bit odd for me. Maybe "Pomiń pytania, na które już znasz odpowiedź"? Then we'd have to make the popup a bit wider